src/fragments/ui-legacy/auth/react/tutorial.mdx

```diff
@@ -60,7 +60,7 @@ yarn add aws-amplify aws-amplify-react
 If integrating with a React Native app, use:
 
 ```
-yarn add aws-amplify aws-amplify-react-native @react-native-picker/picker
+yarn add aws-amplify aws-amplify-react-native
```

Comment:
What is ui-legacy? Why do we leave aws-amplify-react-native here but not in the overview file?

Reply:
It's still there, needs re-routing